src/main/java/net/pms/network/HTTPResource.java

```diff
@@ -44,6 +44,7 @@
 	public static final String UNKNOWN_AUDIO_TYPEMIME = "audio/mpeg";
 	public static final String AUDIO_TRANSCODE = "audio/transcode";
 	public static final String VIDEO_TRANSCODE = "video/transcode";
+	public static final String AUDIO_AAC_TYPEMIME = "audio/mp4";
```

Comment:
@Sami32 Because you are typing AAC to the MP4 mime type. AAC is defined in the MPEG-2 standard, just like MP3 is in "MPEG-1".  Are you saying that ```audio/mp4``` is the correct mime-type for ADTS?